.gitignore

```diff
@@ -51,3 +51,5 @@ buck-out/
 */fastlane/report.xml
 */fastlane/Preview.html
 */fastlane/screenshots
+
+.editorconfig
```

Comment:
You don't have to ignore the `.editorconfig`, it's a shared normalization config.